src/Examine.Lucene/Suggest/LuceneSuggestionOptions.cs

```diff
@@ -0,0 +1,49 @@
+using Examine.Suggest;
+using Lucene.Net.Analysis;
+
+namespace Examine.Lucene.Suggest
+{
+    /// <summary>
+    /// Lucene Suggester query time options
+    /// </summary>
+    public class LuceneSuggestionOptions : SuggestionOptions
+    {
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="top">Clamp number of results</param>
+        /// <param name="suggesterName">The name of the Suggester to use</param>
+        /// <param name="analyzer">Query time Analyzer</param>
+        /// <param name="suggestionMode">Suggestion Mode</param>
+        public LuceneSuggestionOptions(int top = 5, string suggesterName = null, SuggestMode suggestionMode = default) : base(top, suggesterName)
+        {
+            SuggestionMode = suggestionMode;
+        }
+
+        public SuggestMode SuggestionMode { get; }
```

Comment:
This is public and should therefore have XML documentation - Use `/// <inheritdoc />` where appropriate

Reply:
Done